### ISSUE TITLE
Bug: daysInMonth giving wrong values

### ISSUE THREAD

**Issue by zgusth** [johnpryan/date_utils#5]
I've never had problems with this until now. I couldn't find anything but flutter probably did update DateTime.weekday property, which make this plugin give wrong values.

# Problem
`Utils.daysInMonth(DateTime(2018, 9))` give us days from 26/oct to 29/sep, missing 30/sep to 6/oct.

This plugin calculates the last day by using:
```

var daysAfter = 7 - last.weekday;
var lastToDisplay = last.add(new Duration(days: daysAfter));
```

Weekday property says:
```
The day of the week [monday]..[sunday].

In accordance with ISO 8601
   * a week starts with Monday, which has the value 1.
```

Which means Sunday receives the value of 7, and the formula this plugin uses add 0 days.

Since I didn't saw this problem before and this plugin would be correct if Sunday would be 0, I think flutter team updated it.

# Solution
Basically just transform Sunday into 0 again. I'm using daysInRange manually until this plugin is updated, hopefully soon.

```
var lastWeekday = last.weekday % 7;
var daysAfter = 7 - lastWeekday;
var lastToDisplay = last.add(new Duration(days: daysAfter));
```

**Comment by johnpryan** [comment]
This is fixed by https://github.com/apptreesoftware/date_utils/pull/7 - thanks!
